_posts/2021-06-07-Synchronizing-physical-clocks-among-different-Subnero-modems.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Synchronizing physical clocks among different Subnero modems
+banner : images/banner-pulse-synchronization.jpg
+date:  2021-06-07
+categories: wnc
+thumbnail: images/pulse-thumbnail-modems.jpg
+---
+
+All Subnero modems have a “real-time clock” to keep track of time using a precise quartz crystal. Based on the defined frequency of this quartz, software running on the modem can precisely monitor the internal time. When the modem connects to the network (Internet), the software running on the modem then contacts a timer server, which is equipped with a UTC receiver or an accurate clock, to accurately reset the local timer using Network Time Protocol, also called NTP.
```

Comment:
I did not intend to talk about NTP actually. I have removed this. The explanation you are asking about would be the difference between `phy.rtc` and `phy.time`. Could you or @notthetup help add that small paragraph if needed or modify the existing one since I am not aware accurately of the internals of which clock is used where?

Reply:
As part of addressing another comment, I have detailed measurements of `phy.rtc` and `phy.time` and how user can calibrate based on this. Please review and see if that addresses your point.

Reply:
Looks good. Am wondering whether we should add the caveat that if the RTC is not synced to an NTP, the `phy.rtc.getTime()` will give wrong time. Either way, I am OK.